environment/README.md

```diff
@@ -1,77 +1,132 @@
+## Getting the source code
 
-3. Run `cd environment`.
+```bash
+git clone https://github.com/greenelab/ccc.git
+cd ccc/
+```
 
-4. (optional) Adjust your environment variables:
+## Adjusting settings
 
-    ```bash
-    # (optional, will default to subfolder 'cm_gene_expr' under the system's temporary directory)
-    # Root directory where all data will be downloaded to
-    export CM_ROOT_DIR=/tmp/cm_gene_expr
+Adjust the paths where the data will be downloaded, the number of CPU cores available to run the code, etc.
+You can specify these options using environment variables:
 
-    # (optional, will default to half the number of cores)
-    # Adjust the number of cores available for general tasks
-    export CM_N_JOBS=2
+ ```bash
+ # (optional) Root directory where all data will be downloaded to.
+ # Defaults to subfolder 'ccc_gene_expr' under the system's temporary directory.
+ export CM_ROOT_DIR=/tmp/ccc_gene_expr
 
-    # (optional)
-    # Export this variable if you downloaded the manuscript sources and want to
-    # generate the figures for it
-    export CM_MANUSCRIPT_DIR=/tmp/manuscript
-    ```
+ # (optional) Adjust the number of cores available for general tasks.
+ # Defaults to 1 CPU core.
+ export CM_N_JOBS=2
 
-5. (optional) Adjust other settings (i.e. root directory, available computational
-   resources, etc.) by modifying the file `../libs/clustermatch/settings.py`
+ # (optional) Export this variable if you downloaded the manuscript sources
+ # and want to generate the figures for it.
+ export CM_MANUSCRIPT_DIR=/tmp/manuscript
+ ```
 
-6. Adjust your `PYTHONPATH` variable to include the `libs` directory:
+Or you can change these options in `../libs/ccc/settings.py`
 
-    ```bash
-    export PYTHONPATH=`readlink -f ../libs/`:$PYTHONPATH
-    ```
+## Creating a Python environment
 
-    `readlink` might not work on macOS. In that case, simply replace it with
-    the absolute path to the `../libs/` folder.
+Now you need a Python environment.
+You have two choices: 1) using Docker (the easiest) or 2) creating your own conda environment.
 
-7. Run `bash scripts/setup_environment.sh`.
-This will create a conda environment and download the data needed to run the analyses.
-It will download `XXX` GB, so it will take a while to finish.
-Keep in mind two things here:
-1) the script requires conda >= 4.9.0 to work;
-if you cannot upgrade, just remove the `--no-capture-output` parameter in the script.
-2) we use conda packages from the `conda-forge` channel;
-if you find any issues with the installation, it is advisible to configure the channel priority of your conda to `strict` if possible (`conda config --set channel_priority strict`).
+### Using Docker
 
-# Developer usage
+This is the easiest approach. [Install Docker](https://docs.docker.com/get-docker/) for your system and just download the image:
 
-These steps are only for developers.
+```bash
+docker pull miltondp/ccc
```

Comment:
I'm not super familiar with docker. So is this command downloading and installing the docker image? No additional steps needed?

Reply:
Yes, that command downloads the image `miltondp/ccc` and no further steps are necessary to prepare the Python environment. You still need to download the data though.

Reply:
I made changes to clarify all these things better.